python/kserve/kserve/constants/constants.py

```diff
@@ -71,3 +71,22 @@
 
 # K8S metadata key constants
 GENERATION = 'generation'
+
+# model input datatypes and numpy datatypes mapping
+DATATYPE_TO_NUMPY = {
```

Comment:
I think we already have the conversion util defined here?
https://github.com/kserve/kserve/blob/master/python/kserve/kserve/utils/numpy_codec.py#L18

Reply:
Yeah. removed!